plugins/email/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/email/rest/MessageDTO.java

```diff
@@ -0,0 +1,174 @@
+package de.unistuttgart.iaas.amyassist.amy.plugin.email.rest;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Date;
+
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.adapters.XmlJavaTypeAdapter;
+
+import de.unistuttgart.iaas.amyassist.amy.utility.rest.Entity;
+import de.unistuttgart.iaas.amyassist.amy.utility.rest.adapter.LocalDateTimeAdapter;
+
+/**
+ * This class defines a message object which is used for handling mails, it represents a single e-mail
+ * 
+ * @author Patrick Singer
+ */
+@XmlRootElement
+public class MessageDTO extends Entity {
+
+	private String from;
+	private String subject;
+	private String content;
+
+	@XmlJavaTypeAdapter(LocalDateTimeAdapter.class)
+	private LocalDateTime sentDate;
+
+	private boolean important;
+	private boolean seen;
+
+	/**
+	 * Constructor needed to parse as JSON
+	 */
+	public MessageDTO() {
+	}
+
+	/**
+	 * Constructor for a message
+	 * 
+	 * @param from
+	 *            the mail address of the sender
+	 * @param subject
+	 *            subject of the messages
+	 * @param content
+	 *            the content of the message
+	 * @param sentDate
+	 *            the time the mail was sent
+	 * @param important
+	 *            is message from important sender
+	 * @param seen
+	 *            is message seen
+	 */
+	public MessageDTO(String from, String subject, String content, LocalDateTime sentDate, boolean important,
+			boolean seen) {
+		this.from = from;
+		this.subject = subject;
+		this.content = content;
+		this.sentDate = sentDate;
+		this.important = important;
+		this.seen = seen;
+	}
+
+	/**
+	 * Constructor for a message
+	 * 
+	 * @param from
+	 *            the mail address of the sender
+	 * @param subject
+	 *            subject of the messages
+	 * @param content
+	 *            the content of the message
+	 * @param sentDate
+	 *            the time the mail was sent
+	 * @param important
+	 *            is message from important sender
+	 * @param seen
+	 *            is message seen
+	 */
+	public MessageDTO(String from, String subject, String content, Date sentDate, boolean important, boolean seen) {
+		this(from, subject, content, sentDate.toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime(), important,
+				seen);
+	}
+
+	/**
+	 * @return address of the sender
+	 */
+	public String getFrom() {
+		return this.from;
+	}
+
+	/**
+	 * @return subject of message
+	 */
+	public String getSubject() {
+		return this.subject;
+	}
+
+	/**
+	 * @return return content of message
+	 */
+	public String getContent() {
+		return this.content;
+	}
+
+	/**
+	 * @return date when the message was sent
+	 */
+	public LocalDateTime getSentDate() {
+		return this.sentDate;
+	}
+
+	/**
+	 * @return if the message is important
+	 */
+	public boolean isImportant() {
+		return this.important;
+	}
+
+	/**
+	 * @return if the message was seen
+	 */
+	public boolean isSeen() {
+		return this.seen;
+	}
+
+	/**
+	 * Needed for rest test
```

Comment:
if you want to test the rest interface don't rely on this method, this is part of the SUT and should be tested. Test related code must stay in the test resources.